packages/protocol/scripts/bash/verify-deployed.sh

```diff
@@ -32,4 +32,31 @@ done
 source scripts/bash/release-lib.sh
 build_tag $BRANCH $LOG_FILE
 
+if [ "$BRANCH" = "core-contracts.v10" ]; then
+  if [ ! -d "build/mento" ]; then
+    mkdir -p build/mento
+    cd build/mento
+    git clone --depth 1 --branch v2.2.1 https://github.com/mento-protocol/mento-core.git
+    cd mento-core
+    yarn 
+    forge install
+    forge build
```

Comment:
Should this (and the checkout of the appropriate branch) be done whether or not the directory exists?